example/pom.xml

```diff
@@ -25,8 +25,8 @@
 		</dependency>
 		<!-- Includes missing javax.jms.JMSExceptions class -->
```

Comment:
I guess this comment is still valid for the new dep of jakarta, right?

Reply:
Jup 😄